File: modl 4/C/PRAK402-2310817210027-MuhammadErzaRaihan.c
#include <stdio.h>

int main() {
    int Min = 1;
    int Max;
    scanf("%d", &Max);

    for (int Z = Min; Z <= Max; Z++) {
        if (Z % 2 != 0) {
            printf("%d ", Z);
        }
    }

    printf("\n");

    for (int Z = Max; Z >= Min; Z--) {
        if (Z % 2 == 0) {
            printf("%d ", Z);
        }
    }

    printf("\n");
    return 0;
}

File: modl 4/python/PRAK402-2310817210027-MuhammadErzaRaihan.py
Min = 1
Max = int(input(""))

for Z in range(Min , Max +1):
  if Z % 2 == 0:
   print('' , end='')
  else :
    print(Z,'', end='')

print('')

for Z in range(Max,Min, -1):
  if Z % 2 == 0:
    print(Z,'', end='')
 